.github/workflows/ci.yml

```diff
@@ -157,7 +157,7 @@ jobs:
       - name: Install test dependencies
         run: ./ci/install_test_deps.sh.cmd
       - name: Install paddle
-        run: python -m pip install paddlepaddle
+        run: python -m pip install paddlepaddle gast==0.3.3
```

Comment:
Why gast here?

Reply:
apparently when install `paddlepaddle` from pypi it install gast==0.5.5, which ran into errors in CI. `paddlepaddle` requires gast to be >=0.3.3 and <0.4.0, but I'm not entirely sure why it install `gast==0.5.5`, so thats why we install it there.

Reply:
I see.

> but I'm not entirely sure why it install gast==0.5.5
Oh that's a known behavior of pip. BTW that's why we need dependency locking when building bento bundle.